odooly.py

```diff
@@ -1455,7 +1455,7 @@ def clone_database(self, passwd, database, neutralize_database=False):
         self.env._cache_set('auth', dict(auth_cache), db_name=database)
 
         # Login with the current user into the new database
-        (uid, password, __) = self.env._auth(self.env.uid, None)
+        (uid, password, __) = self.env._auth(self.env.uid, None, None)
```

Comment:
Good catch. I did not see it, and tests are missing this one.

Maybe it should be `self.env._api_key` here:
```suggestion
        (uid, password, __) = self.env._auth(self.env.uid, None, self.env._api_key)
```

I suppose that the API Key is still valid in the new database.

Reply:
Hello, I have made this change in the PR because it makes sense to me, but I do not really have an environment set up to test with api_key. I can only confirm that it works correctly with the password (using /jsonrpc).

Reply:
I'll merge and refactor.

This part does not need to call `_auth` actually.
Purpose was to retrieve the password in cache in order to login to the new DB.
It can be done in a simpler way.